src/components/ChatEntry.js

```diff
@@ -1,22 +1,45 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js'
+
 
 const ChatEntry = (props) => {
+
+  const onLikeButtonClick = () => {
+    const updatedChatEntry = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+    props.onUpdate(updatedChatEntry);
+  }
```

Comment:
Nice callback! We could DRY this up a little by making a copy and changing the `liked` key to the reverse: 
```
const updatedChatEntry = {...props}
delete updatedChatEntry[onUpdate];
updatedChatEntry.liked = !props.liked
```